CHANGELOG.md

```diff
@@ -36,4 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#1](https://github.com/DFE-Digital/polis-whitelabel/issues/1) Fixed `authorUid is undefined` crash in `server`
 - [#3](https://github.com/DFE-Digital/polis-whitelabel/issues/3) Fixed crash when `.headersJson` file not present for `file-server`
 
+### Performance
```

Comment:
Good shout - I've had this problem as well - maybe make it a 'Refactor' title to encompass this and some other similarly not-really-bugs-or-features?